packaging/ios/Subsurface-mobile.pro

```diff
@@ -81,6 +81,7 @@ SOURCES += ../../subsurface-mobile-main.cpp \
 	../../core/btdiscovery.cpp \
 	../../core/connectionlistmodel.cpp \
 	../../core/qt-ble.cpp \
+	../../core/save-profiledata.c \
```

Comment:
that should be done earlier in the series

Reply:
yes